config/locales/views/assignments/en.yml

```diff
@@ -73,4 +73,5 @@ en:
       started_on: 'You started on:'
       starter_file_prompt: Click below to download the starter files for this assessment.
       time_until_due_warning: This assessment is due before %{due_date} without penalty.
+    upload_config_help_html: ' This zip file will configure and create this assignment.<br> For more information please visit <a href="https://github.com/MarkUsProject/Markus/wiki/Instructor-Guide--Importing-and-Exporting-Data">this page</a>.'
```

Comment:
Please remove the whitespace at the beginning of the string.

Reply:
This is good but we should make sure to actually include information about this in the wiki. You can add proposed changes to the wiki by editing this file https://github.com/MarkUsProject/Markus/wiki/Pending-Changes.

You can edit that file to describe the changes that you would make to the wiki including:

- which page will to changes go (in this case it's the "Instructor-Guide--Importing-and-Exporting-Data" page)
- where on the page will you add the new information
- what information will you add/change/delete etc.
- which pull request this relates to

When you've done that, update the link here to reference the precise section on the page that you have added so when users click on it, it will take them to the precise location on the page with the relevant information. 

When we merge these changes into production we will then add the content of the Pending Changes page to actual wiki.